### PR TITLE
Add www.browserscope.org

### PR DESCRIPTION
**Pull request by mathiasbynens**
Browserscope is available over HTTPS as of yesterday!

### CODE REVIEW

**Comment by mathiasbynens** [comment]
Build failure seems to be unrelated to this change. :/

File: src/chrome/content/rules/www.browserscope.org.xml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+<ruleset name="www.browserscope.org">
+	<target host="www.browserscope.org" />
+	<rule from="^http:" to="https:" />
+	<securecookie host="^www\.browserscope\.com$" name=".*" />
```
Comment (d10) [inline src/chrome/content/rules/www.browserscope.org.xml]: Should this be `.org`?

Comment (mathiasbynens) [inline src/chrome/content/rules/www.browserscope.org.xml]: Good catch, thanks. Fixed!

**Comment by semenko** [comment]
Awesome! Thanks!

**Pull request merged**